CHANGELOG.md

```diff
@@ -3,7 +3,6 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) though uses different version number scheme (YY.MM.DD.NN) where NN is build number.
 
-## Unreleased
-
-### Added
- - workflow mode to allow users to execute workflows 
+## [20.04.10.00] - 2020-04-10
```

Comment:
Lets keep unreleased for now, while we figure out how we do release numbers in new world ;)